.env.example

```
@@ -19,6 +19,10 @@ WORKOS_REDIRECT_URI=http://localhost:3000/auth/callback
 # Session signing secret (generate: openssl rand -hex 32)
 SESSION_SECRET=
 
+# Transactional email (Resend)
+RESEND_API_KEY=
+EMAIL_FROM=noreply@rustume.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_FROM key should go before the RESEND_API_KEY key

(UnorderedKey)